tests/src/test/scala/scalaz/DListTest.scala

```diff
@@ -38,7 +38,7 @@ object DListTest extends SpecLite {
     def monoid[A] = Monoid[DList[A]]
     def monadPlus = MonadPlus[DList]
     def alt = Alt[DList]
-    def bindrec = BindRec[DList]
+    def bindRec = BindRec[DList]
```

Comment:
This change is the scariest (it occurs in three files).

I can't understand how this code could compile in both forms. Based on the surrounding code, this change appears to be correct, but 🤷 .

Reply:
It's just a unused method to test whether `BindRec[DList]` compiles or not. So any name would be OK.